src/Rings/orderings.jl

```diff
@@ -124,7 +124,7 @@ function _unique_var_indices(a::AbstractVector{<:MPolyRingElem})
   return z
 end
 
-function isweighted(ord::Symbol)
+function is_weighted(ord::Symbol)
```

Comment:
I find this method problematic: in how far is a *symbol* weighted? This is perhaps OK for an internal helper, but not for something we export. I would suggest to call this `_is_weighted`

Reply:
I have the impression that this function should be removed completely. It is not used (apparently it was used originally for a `show` method) and monomial orderings should not be represented by symbols nowadays. So if the functionality is needed, there should be a `is_weighted(::MonomialOrdering)` in my opinion.

It looks like one could easily implement this with
```
is_weighted(::MonomialOrdering) = false
is_weighted(::WSymbOrdering) = true
```
but no guarantee.

Reply:
The reason why I opened this PR is that I need this for #2936. Users should be able to provide an ordering via symbol as an input there. So please don't remove